package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dynamoose",
-  "version": "2.4.0",
+  "version": "2.5.0",
```

Comment:
I see. I'll change the version back to 2.4.0. No reason for the minor version update just I am thinking this is a non-leaky version of dynamoose and that means it's a big change over existing versions of dynamoose. But 2.4.1 makes sense too.

Reply:
I'll also add more details to my pr. We were looking at the memory leaks in production and realized because we compile to es6 and there would be generators with syntaxes `__awaiter` in the code. Once we compile to es2017 the code doesn't have those syntaxes anymore and our memory leak issue is gone.

Reply:
@shihminlee Got it. Not sure quite what you mean by `non-leaky version`. It is a big change, but I don't think it's a breaking change. It still works on Node.js >8. The code base is the same. All of the tests still pass fine. So I do see the argument for it being a minor version.

However I don't think there is any solid evidence at this point that it deserves a minor version. It's all theoretical.

Just wanted to make sure there wasn't anything I was missing. There is a chance I'll include this in 2.5.0 anyways. Kinda depends on the timeline for how quickly you think this should be pushed out. Seems like a big enough deal to warrant a release with just this in it. But it might be good to include it with some other stuff and just do it in 2.5.0.

Reply:
Sure. a patch version is reasonable too.